<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDecorationTileGridDialog</class>
 <widget class="QDialog" name="QgsDecorationTileGridDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tile Grid Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="grpEnable">
     <property name="title">
      <string>Enable Tile Grid</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QComboBox" name="mGridTypeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mGridTypeLabel">
        <property name="text">
         <string>Grid type</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QGroupBox" name="grpStaticDynamic">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="mDynamicGrid">
           <property name="text">
            <string>Dynamic Grid</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="dynamicGroupBox">
           <property name="title">
            <string/>
           </property>
           <widget class="QLabel" name="mZoomFactorLabel">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>20</y>
              <width>251</width>
              <height>33</height>
             </rect>
            </property>
            <property name="text">
             <string>Zoom Factor</string>
            </property>
           </widget>
           <widget class="QSlider" name="mZoomFactorSlider">
            <property name="geometry">
             <rect>
              <x>410</x>
              <y>20</y>
              <width>160</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="mStaticGrid">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Static Grid</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="staticGroupBox">
           <property name="title">
            <string/>
           </property>
           <widget class="QLabel" name="mZoomLevelLabel">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>0</y>
              <width>241</width>
              <height>64</height>
             </rect>
            </property>
            <property name="maximumSize">
             <size>
              <width>583</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Zoom Level</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mZoomLevelSpinBox</cstring>
            </property>
           </widget>
           <widget class="QSpinBox" name="mZoomLevelSpinBox">
            <property name="geometry">
             <rect>
              <x>480</x>
              <y>10</y>
              <width>89</width>
              <height>48</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>22</number>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="22" column="1">
       <widget class="QgsFontButton" name="mAnnotationFontButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="23" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mMarkerSymbolLabel">
        <property name="text">
         <string>Marker symbol</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QRadioButton" name="mCoordinatesNone">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Do not show coordinates</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupCoordinatesDisplay</string>
        </attribute>
       </widget>
      </item>
      <item row="22" column="0">
       <widget class="QLabel" name="mAnnotationFontLabel">
        <property name="text">
         <string>Coordinates font</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QRadioButton" name="mCoordinatesCenter">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Show coordinates in tile center</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupCoordinatesDisplay</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mLineSymbolLabel">
        <property name="text">
         <string>Line symbol</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsSymbolButton" name="mMarkerSymbolButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsSymbolButton" name="mLineSymbolButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QRadioButton" name="mCoordinatesBorder">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Show coordinates on canvas border</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupCoordinatesDisplay</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFontButton</class>
   <extends>QToolButton</extends>
   <header>qgsfontbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSymbolButton</class>
   <extends>QToolButton</extends>
   <header>qgssymbolbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>grpEnable</tabstop>
  <tabstop>mGridTypeComboBox</tabstop>
  <tabstop>mLineSymbolButton</tabstop>
  <tabstop>mMarkerSymbolButton</tabstop>
  <tabstop>mDynamicGrid</tabstop>
  <tabstop>mZoomFactorSlider</tabstop>
  <tabstop>mStaticGrid</tabstop>
  <tabstop>mZoomLevelSpinBox</tabstop>
  <tabstop>mCoordinatesNone</tabstop>
  <tabstop>mCoordinatesCenter</tabstop>
  <tabstop>mCoordinatesBorder</tabstop>
  <tabstop>mAnnotationFontButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroupCoordinatesDisplay"/>
 </buttongroups>
</ui>
